src/sourmash/tax/__main__.py

```diff
@@ -0,0 +1,300 @@
+"""
+Command-line entry point for 'python -m sourmash.tax'
+"""
+import sys
+import csv
+import json
```

Comment:
this is imported but unused.

(please run `pyflakes src/sourmash/tax` and fix all the relevant warnings; there's a lot of unused stuff in here)

Reply:
done! sidenote, hadn't done this before and it was super neat and helpful

Reply:
right? I have started to use it more habitually ;).